book/website/community-handbook/governance/gov-constitutional.md

```diff
@@ -0,0 +1,131 @@
+(ch-gov-constitutional)=
+# Consitutional level decision making
+
+Decisions taken at the consitutional level affect multiple working groups and delivery groups and therefore a broad and diverse group of individual community members.
+They will usually have long term implications for members of *The Turing Way* community.
+Decisions taken at the consitutional level may require a prioritisation of activities, for example for the Community Management delivery groups(s).
+
+Decisions at this level are made by the *The Turing Way*'s Steering Council.
+
+## Membership
+
+The Steering Council is made up of the chairs of the working groups and delivery groups.
+This number is not limited.
+
+### Electing a chair 
+
+The Steering Council elects a chair from their membership.
+The responsibilities of the chair are detailed in the <!-- MAKE INTO LINK-->role of a chair page in this chapter.
+
+The Chair of the Steering Council holds their role for a calendar year, starting in January and handing over their responsibilities in December.
+
+The Chair joins as Chair Elect for one year, holds the role of Chair for one year, and remains on the Steering Council for one year as Past Chair.
```

Comment:
I like the idea that a chair is involved before and after their term. Hopefully it will help smooth the transitions and mean we can commit to long term goals spanning multiple terms.